caluma/extensions/tests/test_events.py

```diff
@@ -353,7 +353,7 @@ def test_send_work_item_mail(  # noqa: PLR0915
         ]
     case = document_review_case
     if early_career_form:
-        case.document.form.slug = settings.EARLY_CAREER_AWARD_FORM_SLUG
+        case.document.form.slug = settings.EARLY_CAREER_AWARD_FORM_SLUGS[0]
```

Comment:
I was just about to complain about that missing `S` :smile_cat:

Reply:
CI already did that for you :wink: